code/datums/status_effects/neutral.dm

```diff
@@ -47,25 +47,101 @@
 
 /datum/status_effect/high_five
 	id = "high_five"
-	duration = 5 SECONDS
+	duration = 10 SECONDS
 	alert_type = null
+	status_type = STATUS_EFFECT_REFRESH
+	/// Message displayed when wizards perform this together
+	var/critical_success = "high-five EPICALLY!"
+	/// Message displayed when normal people perform this together
+	var/success = "high-five!"
+	/// Message displayed when this status effect is applied.
+	var/request = "requests a high-five."
+	/// Item to be shown in the pop-up balloon.
+	var/obj/item/item_path = /obj/item/latexballon
+	/// Sound effect played when this emote is completed.
+	var/sound_effect = 'sound/weapons/slap.ogg'
+
+/// So we don't leave folks with god-mode
+/datum/status_effect/high_five/proc/wiz_cleanup(mob/user, mob/highfived)
+	user.status_flags &= ~GODMODE
+	highfived.status_flags &= ~GODMODE
+
+/datum/status_effect/high_five/on_apply()
+	if(!iscarbon(owner))
+		return FALSE
+	. = ..()
+
+	var/mob/living/carbon/user = owner
+	var/is_wiz = iswizard(user)
+	for(var/mob/living/carbon/C in orange(1, user))
+		if(!C.has_status_effect(type) || C == user)
+			continue
+		if(is_wiz && iswizard(C))
+			user.visible_message("<span class='biggerdanger'><b>[user.name]</b> and <b>[C.name]</b> [critical_success]</span>")
+			user.status_flags |= GODMODE
+			C.status_flags |= GODMODE
+			explosion(get_turf(user), 5, 2, 1, 3, cause = id)
+			// explosions have a spawn so this makes sure that we don't get gibbed
+			addtimer(CALLBACK(src, PROC_REF(wiz_cleanup), user, C), 1)
+			add_attack_logs(user, C, "caused a wizard [id] explosion")
+			user.remove_status_effect(type)
+			C.remove_status_effect(type)
+
+		user.do_attack_animation(C, no_effect = TRUE)
+		C.do_attack_animation(user, no_effect = TRUE)
+		user.visible_message("<span class='notice'><b>[user.name]</b> and <b>[C.name]</b> [success]</span>")
+		playsound(user, sound_effect, 80)
+		user.remove_status_effect(type)
+		C.remove_status_effect(type)
+		return FALSE
+
+	owner.custom_emote(EMOTE_VISIBLE, request)
+	// this can only go well
+	var/obj/item/bloon = new item_path()
+	owner.create_point_bubble(bloon)
+	QDEL_IN(bloon, 10)
```

Comment:
Creating an item here to just point at it feels a bit of a hack. I think refactoring the `create_point_bubble` code and extracting the bubble making code out of it will be better in the long term. Else other code also needs to apply this kind of hack in the future.

Reply:
Addressed, added some functions to hopefully make it a little easier to do going forward. Would be cool to see this bubble functionality used in some more clever ways going forward.